src/drt/src/io/io.cpp

```diff
@@ -855,6 +855,27 @@ void updatefrAccessPoint(odb::dbAccessPoint* db_ap,
       }
     }
   }
+  auto db_path_segs = db_ap->getSegments();
+  for(const auto& db_path_seg : db_path_segs) {
+    Rect db_rect = std::get<0>(db_path_seg);
+    bool begin_style_trunc = std::get<1>(db_path_seg);
+    bool end_style_trunc = std::get<2>(db_path_seg);
```

Comment:
@Mahfouz-z  replace with: `for(const auto& [db_rect, begin_style_trunc, end_style_trunc] : db_path_segs)`